Doppler.Sap.Job/Settings/DopplerRepositorySettings.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Doppler.Sap.Job.Service.Settings
+{
+    public class DopplerRepositorySettings
+    {
+        public string ConnectionString { get; set; }
+    }
```

Comment:
Is this class being used?

Reply:
Now yes with this commit https://github.com/MakingSense/doppler-jobs/pull/6/commits/f5d1f949ed4d5c0dff5e14b70938148ea69b8c86